.travis.yml

```diff
@@ -1,6 +1,5 @@
 sudo: false
 language: node_js
 node_js:
-  - 4
   - 6
   - 7
```

Comment:
While you are changing this file - please remove `6` and `7` from the list (these Node.js version are no longer supported by the Node.js project) and add `8`, `10` and `12`.